src/stores/b2b-purchase-order-flow.md

```diff
@@ -0,0 +1,82 @@
+---
+b2b_only: true
+title: Purchase order workflow
+group: getting-started
+---
+
+Purchase orders allow companies to track and control spending. When purchase orders are enabled for a company account, all orders are automatically processed as purchase orders.
+
+The purchase order workflow can vary in a few ways:
+
+- If approval rules are established, regular users will go through the approval process.
+- Company admins can bypass the approval rules.
+- Offline payment details are entered when creating the purchase order.
+- Online payment details are entered after the purchase order has been approved.
+
+This topic will cover basic concepts and workflows for purchase orders.
+
+## Basic purchase order workflow
+
+Companies use purchase orders to control which employees can purchase on behalf of the company and often set up approval rules to enforce company guidelines. Depending on the approval rules, multiple people might have to approve the order.
```

Comment:
>Companies use purchase orders to control which employees can purchase

should this be: `Companies use purchase orders to control WHAT employees can purchase`?

Reply:
Excellent point.